portal/src/locale-data/en.json

```diff
@@ -1577,6 +1577,9 @@
   "SMSProviderConfigurationScreen.provider.twilio": "Twilio",
   "SMSProviderConfigurationScreen.provider.webhook": "Webhook",
   "SMSProviderConfigurationScreen.provider.deno": "Custom JS/TS",
+  "SMSProviderConfigurationScreen.form.twilio.twilioSID": "Application SID",
```

Comment:
Reword the label to `Twilio Account SID` This is the official name, and people can google [https://help.twilio.com/articles/14726256820123-What-is-a-Twilio-Account-SID-and-where-can-I-find-it-](https://help.twilio.com/articles/14726256820123-What-is-a-Twilio-Account-SID-and-where-can-I-find-it-)